README.md

```diff
@@ -9,26 +9,26 @@ For more details about the design and architecture of the SDK, see the [official
 ### Required Prerequisites
 To use this SDK you must have:
 
-* **A Java 8 development environment**
+* **A Java 8 or newer development environment**
 
-  If you do not have one, go to [Java SE Downloads](https://www.oracle.com/technetwork/java/javase/downloads/index.html) on the Oracle website, then download and install the Java SE Development Kit (JDK). Java 8 or higher is required.
+  If you do not have one, we recommend [Amazon Corretto](https://aws.amazon.com/corretto/).
 
   **Note:** If you use the Oracle JDK, you must also download and install the [Java Cryptography Extension (JCE) Unlimited Strength Jurisdiction Policy Files](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html).
 
-* **Bouncy Castle**
+* **Bouncy Castle** or **Bouncy Castle FIPS**
 
-  Bouncy Castle provides a cryptography API for Java. If you do not have Bouncy Castle, go to https://bouncycastle.org/latest_releases.html, then download the provider file that corresponds to your JDK. Or, you can pick it up from Maven:
+  Bouncy Castle provides some utilities the AWS Encryption SDK needs to handle cryptographic objects.
+  If you do not have Bouncy Castle, go to https://bouncycastle.org/latest_releases.html, then download the provider file that corresponds to your JDK.
+  Or, you can pick it up from Maven (groupId: `org.bouncycastle`, artifactId: `bcprov-ext-jdk15on`).
 
-  ```xml
-  <dependency>
-    <groupId>org.bouncycastle</groupId>
-    <artifactId>bcprov-ext-jdk15on</artifactId>
-    <version>1.61</version>
-  </dependency>
-  ```
+  As of version 1.6.1,
+  the AWS Encryption SDK will also work with Bouncy Castle FIPS (groupId: `org.bouncycastle`, artifactId: `bc-fips`).
+  Please follow Bouncy Castle's FIPS [documentation](https://www.bouncycastle.org/documentation.html),
+  including User Guides and Security Policy for proper installation and configuration.
```

Comment:
```suggestion
For help installing and configuring Bouncy Castle FIPS, see Bouncy Castle's [BC FIPS Documentation](https://www.bouncycastle.org/documentation.html),
  including the User Guides and Security Policy.
```

Reply:
I moved the end of the sentence to the beginning, so we can remove the end ("for proper installation and configuration"). Otherwise, it's redundant. If you want the "proper" in there, put it in the opening clause.